crates/bevy_ecs/src/schedule/executor_parallel.rs

```diff
@@ -193,7 +193,10 @@ impl ParallelExecutor {
                         .unwrap_or_else(|error| unreachable!("{}", error));
                     #[cfg(feature = "trace")]
                     let system_guard = system_span.enter();
-                    unsafe { system.run_unsafe((), world) };
+                    #[allow(clippy::undocumented_unsafe_blocks)] // TODO: document why this is safe
```

Comment:
```suggestion
                    // SAFETY: the executor prevents two systems with conflicting access from running simultaneously.
```

Reply:
it's the call to this function https://github.com/bevyengine/bevy/blob/main/crates/bevy_ecs/src/schedule/executor_parallel.rs#L244